MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxMaps", "~> 10.10"
+  s.dependency "MapboxMaps", "~> 10.10.1"
```

Comment:
Same here:
```suggestion
  s.dependency "MapboxMaps", "~> 10.10"
```

Reply:
If we use `~> 10.10` in the `podspec`, the CocoaPod dependency will be `10.10` for `CocoaPodsTest`

Reply:
That's strange, I've just tried with default podspec where `s.dependency "MapboxMaps", "~> 10.10"` is used and the most recent version was resolved in Podfile.lock (10.10.1).

Reply:
Thanks for the suggestion, issue was solved.